packages/mobile/src/components/details-tile/MetadataRow.tsx

```diff
@@ -0,0 +1,34 @@
+import type { PropsWithChildren } from 'react'
+
+import type { Mood } from '@audius/sdk'
+import { Image } from 'react-native'
+
+import { Flex, Text, spacing } from '@audius/harmony-native'
+import { moodMap } from 'app/utils/moods'
+
+type MetadataRowProps = PropsWithChildren<{
+  label: string
+}>
+
+export const MetadataRow = ({ label, children }: MetadataRowProps) => {
+  return (
+    <Flex direction='row' alignItems='center' key={label} gap='xs'>
+      <Text variant='label'>{label}</Text>
+      {label === 'Mood' ? (
+        <Flex direction='row' gap='xs' alignItems='center'>
+          <Text variant='body' size='s' strength='strong'>
+            {children}
+          </Text>
+          <Image
+            source={moodMap[children as Mood]}
```

Comment:
It's a little weird, but I get that typing the output of the new hook might be more difficult than it's worth

Reply:
ohhhh wait i meant to split this out, forgot this spot.